composer.json

```diff
@@ -77,5 +67,15 @@
         "symfony": {
             "allow-contrib": false
         }
-    }
+    },
+    "repositories": [
+        {
+            "type": "composer",
```

Comment:
You should remove this and rebase so that it disappears from git history! (Albanil is not public)